api_tests/actions/views/test_schema_response_action_list.py

```diff
@@ -68,6 +68,13 @@ def configure_test_preconditions(
         schema_response.pending_approvers.add(
             *[user for user, _ in registration.get_admin_contributors_recursive()]
         )
+
+    if schema_response_state is ApprovalStates.IN_PROGRESS:
+        # need valid changes for submission validations
+        schema_response.update_responses({'q1': 'update for submission'})
```

Comment:
we're setting the responses above (L62), so this shouldn't be needed, but it's also not breaking anything, so is fine for the moment